core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/console/CodeSystemsCommand.java

```diff
@@ -57,6 +59,9 @@
 public final class CodeSystemsCommand extends Command {
 
 	private static final Ordering<CodeSystem> SHORT_NAME_ORDERING = Ordering.natural().onResultOf(CodeSystem::getShortName);
+	
+	private static final String CODE_SYSTEM_INFO_JOIN_DELIMITER = "\n   ";
```

Comment:
Having spaces here is fine, however if you use tabulation it will be immediately visible for anyone who reads the code what is the difference between the different indentations. E.g.:
```suggestion
	private static final String CODE_SYSTEM_INFO_JOIN_DELIMITER = "\n\t";
```

Reply:
Done